internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -725,6 +725,42 @@ func (s *LoadBalancedWebService) convertGracePeriod() *int64 {
 	return aws.Int64(int64(manifest.DefaultHealthCheckGracePeriod))
 }
 
+func (s *LoadBalancedWebService) convertImportedALB() (*template.ImportedALB, error) {
+	if s.importedALB == nil {
+		return nil, nil
+	}
+	l := s.importedALB.Listeners
+	var twoListeners bool
+	if len(l) == 2 {
```

Comment:
+1 to wx's fb above. can we error out if they have more than 2 listeners for now? Since we won't be handle properly for this case.